### PR TITLE
[policy][doc enhancement][1462062]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Updated the content. Please review

### CODE REVIEW

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 1]
```diff
@@ -17,46 +16,64 @@ This article addresses common reasons submitted apps fail validation. While it's
...
 ### &#9989; General considerations
 
-*See also* [Section 100 — General](/legal/marketplace/certification-policies#100-general)
+See also [Section 100 — General](/legal/marketplace/certification-policies#100-general)
 
 * Ensure you are using version 1.4.1 or later of the [Microsoft Teams SDK](https://www.npmjs.com/package/@microsoft/teams-js).
 * Don't make changes to your app while the validation process is in progress. Doing so will require a complete revalidation of your app.
-* Your app  must not stop responding, end unexpectedly, or contain programming errors. If an issue is encountered, your app should fail gracefully and provide a valid-way-forward message to the user.
-* Your app must not automatically download, install, or launch any executable code in the user environment. All downloads should seek explicit permission from the user.
+* Your app must not stop responding, end unexpectedly, or contain programming errors. If an issue occurs, your app must fail and provide valid way-forward information to the user.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:23]:
```Suggestion
* Your app must not stop responding, end unexpectedly, or contain programming errors. If an issue occurs, your app must fail and provide valid information for the way-forward to the user.
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 2]
```diff
@@ -17,46 +16,64 @@ This article addresses common reasons submitted apps fail validation. While it's
...
 * Personal apps enable users to share content from a personal app experience with other team members.
 
-### &#9989;  Provide a clear and simple sign in/sign out and sign-up experience
+### &#9989; Provide a clear and simple sign-in, sign-out, and sign-up experience
 
-*See also* [Section 1100.5 — Customer control](/legal/marketplace/certification-policies#11005-customer-control)
+See also [Section 1100.5 — Customer control](/legal/marketplace/certification-policies#11005-customer-control)
 
-* If your app or add-in depends on external accounts or services, the sign in/sign out and sign-up experience must be apparent and reachable across all capabilities in your app.
+* If your app or add-in depends on external accounts or services, the sign-in, sign-out and sign-up experience must be apparent and reachable across all capabilities in your app.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:54]: Add the missing leading comma.
```suggestion
* If your app or add-in depends on external accounts or services, the sign-in, sign-out, and sign-up experience must be apparent and reachable across all capabilities in your app.
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 3]
```diff
@@ -17,46 +16,64 @@ This article addresses common reasons submitted apps fail validation. While it's
...
   * Connect an account/service => disconnect an account/service.
   * Authorize an account/service => deauthorize/deny an account/service.
   * Register an account/service => deregister/unsubscribe an account/service.
-* If your app requires an account or service, you must provide a way for the user to sign up or to create a sign-up request. An exception may be granted if your app requires a license to use. But such scenarios, a clear way forward for a new user sign up must be provided.
-* Make sure you provide clear-way-forward guidance to a new user on how to sign up to use your app services. If a ready sign-up link is not available, a clear way forward may be provided in the following areas
+* If your app requires an account or service, you must provide a way for the user to sign-up or create a sign-up request. An exception may be granted if your app requires a license to use. In such scenarios, provide clear instructions for a new user to sign-up.
+* Provide clear way-forward guidance to a new user on how to sign-up to use your app services. If a ready sign-up link is not available, provide precise guidance in the following areas:
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:66]:
```suggestion
* Provide clear guidance on the way-forward to a new user on how to sign-up to use your app services. If a ready sign-up link is not available, provide precise guidance in the following areas:
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 4]
```diff
@@ -121,23 +143,30 @@ Your bot should be responsive to any command and not dead-end the user. Here are
...
 
 * **Think through all scopes**. Be sure that your bot provides appropriate responses when mentioned (`@*botname*`) in a channel and in personal conversations. If your bot does not provide meaningful context within the personal or teams scope, disable that scope via the manifest. (See the `bots` block in the [Microsoft Teams manifest schema reference](../../../../resources/schema/manifest-schema.md#bots).)
 
-* **Include team, group chat, or 1:1 conversation**. Bot notifications should include a Team, a group chat, or a one-to-one conversation with relevant content for your audience.
+* **Include team, group chat, or 1:1 conversation**. Bot notifications must include a team, a group chat, or a one-to-one conversation with relevant content for your audience.
 
-* **Do not push sensitive data**. Bots must not push sensitive data to a Team, a group chat or a 1:1 conversation where there is an audience who should not be able to view that data
+* **Do not push sensitive data**. Bots must not push sensitive data to a team, a group chat or a 1:1 conversation where there is an audience who should not be able to view that data.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:162]: missing leading comma

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 5]
```diff
@@ -121,23 +143,30 @@ Your bot should be responsive to any command and not dead-end the user. Here are
...
 
 * **Provide a welcome message**. Bot must provide an FRE welcome message that includes an interactive tutorial with carousel cards or "try it" buttons, to encourage engagement.
 
 ### &#9989; Personal bots must always send a welcome message on first launch
 
-A welcome messages is the best way to set the tone for your personal/chat bot. This is the first interaction a user has with the bot. A good welcome message can encourage the user to keep exploring the app. If the welcome or introductory message is confusing or unclear, users won't see the value of the app immediately and lose interests.
-Please see below section for Welcome message requirements.
+A welcome messages is the best way to set the tone for your personal chat bot. This is the first interaction a user has with the bot. A good welcome message can encourage the user to keep exploring the app. If the welcome or introductory message is confusing or unclear, users won't see the value of the app immediately and lose interest.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:168]:
```suggestion
A welcome message is the best way to set the tone for your personal chat bot. This is the first interaction a user has with the bot. A good welcome message can encourage the user to keep exploring the app. If the welcome or introductory message is confusing or unclear, users won't see the value of the app immediately and lose interest.
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 6]
```diff
@@ -121,23 +143,30 @@ Your bot should be responsive to any command and not dead-end the user. Here are
...
+See the following section for welcome message requirements.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:169]:
```suggestion
See the following section for welcome message requirements:
```

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 7]
```diff
@@ -167,12 +196,12 @@ Notification-only bots must send a welcome message that includes a message conve
 
 #### Welcome messages in the personal scope
 
-* **Make your message concise and informative**.  Most likely, user experience with and knowledge of your app will vary. A user may have used your app on another platform or know nothing about your app. You want to tailor your message to all audiences and in a couple sentences explain what your bot does and the ways to interact with it. You should also explain the value of the app and how the users will benefit from using it.
+* **Make your message concise and informative**.  Most likely, user experience with and knowledge of your app will vary. A user may have used your app on another platform or know nothing about your app. You want to tailor your message to all audiences and in a couple sentences explain what your bot does and the ways to interact with it. You must also explain the value of the app and how the users will benefit from using it.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:199]: Please rephrase "Most likely, user experience with and knowledge of your app will vary."

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - verify and commit the suggestions. Incorporate the review comments. Add the SME and Surbhi Gupta as reviewers.

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 8]
```diff
@@ -17,46 +16,64 @@ This article addresses common reasons submitted apps fail validation. While it's
...
+
+* The visual tour can be a video, screenshot, a link to a static tab with app details. All these references must be within the Teams environment.
+
+    ![Sample App 1](../../../../assets/images/faq/Sampleapp1.png)
+    ![Sample App 2](../../../../assets/images/faq/Sampleapp2.png)
+
 * Increment your app version number in the manifest if you make any manifest changes to your submission.
-* App must not take users out of Teams for core user scenarios. Link targets in apps must not link to an external browser but should link to div elements contained within Teams e.g. inside Task Modules and tabs.
+* The app must not take users out of Teams for core user scenarios. Link targets in apps must not link to an external browser. Link targets must link to div elements contained within Teams, for example, inside task modules and tabs.
+* All core and non-core scenarios must be completed within the Teams environment except for:
```
Comment (SonalNigam28) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md:42]: The point at line number 41 and line number 27 seems to be repetitive and a subset of line number 42. Can we merge them all.

**Review by SonalNigam28 — CHANGES_REQUESTED** [review]
Added few changes, kindly fix  them before merging

**Review by SonalNigam28 — APPROVED** [review]
Looks good . please go ahead

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**